gulpfile.js

```diff
@@ -39,6 +39,10 @@ function getServiceNameFromOutputFolderValue(outputFolderValue) {
   return outputFolderSegments[outputFolderSegments.length - 1];
 }
 
+function npmInstall(packageFolderPath) {
+  execSync(`npm install`, { cwd: packageFolderPath, stdio: ['ignore', 'ignore', 'pipe'] });
```

Comment:
does this make it so we only see errors from the child process?

Reply:
Yes.